client/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "noEmit": true,
+    "esModuleInterop": true,
```

Comment:
`noImplicitAny: true` maybe?

Reply:
Added "noImplicitAny". It seems like there are no implicit anys, but it is hard to tell, because it doesn't seem like doing `npm run dev` or `npx tsc` catches them.